kinto/plugins/history/listener.py

```diff
@@ -0,0 +1,94 @@
+from kinto.core.utils import instance_uri
+import copy
+from datetime import datetime
+
+
+def on_resource_changed(event):
```

Comment:
Could we use a Listener there? https://github.com/Kinto/kinto-changes/blob/master/kinto_changes/listener.py#L11